pattern-library/sass/patterns/_alerts.scss

```diff
@@ -0,0 +1,132 @@
+// ------------------------------
+// edX Pattern Library: Utilities - Alerts
+//
+// About: Contains base styling for alerts.
+// ----------------------------
+
+// #CONFIG
+// #UTILITIES
+// #GENERAL
+// #INDIVIDUAL CASES
+
+
+// ----------------------------
+// #CONFIG
+// ----------------------------
+$alert-background-color: palette(grayscale, white) !default;
+$alert-border-radius: rem(3) !default;
+// border around the sides and bottom
+$alert-border: rem(1) solid palette(grayscale-cool, light) !default;
+
+// alert colors
+$alert-information-color: palette(information, base) !default;
+$alert-warning-color: palette(warning, base) !default;
+$alert-error-color: palette(error, base) !default;
+$alert-success-color: palette(success, base) !default;
+
+// ----------------------------
+// #UTILITIES
+// ----------------------------
+@mixin alert($alert-color) {
+    border-top:  rem(2) solid $alert-color;
+
+    .alert-icon {
+        color: palette(grayscale, white);
+        background-color: $alert-color;
+    }
+}
+
+@mixin alert-message($width) {
+
+    @include susy-media($bp-screen-md) {
+        @include float(left);
```

Comment:
Is the float function left-to-right aware? Do we need to do something to make this work right-to-left?